charts/trusted-artifact-signer/README.md

```diff
@@ -137,6 +128,14 @@ Kubernetes: `>= 1.19.0-0`
 | configs.rekor.signer.secret.name | Name of the secret to create with the private key data. This name must match the value in scaffold.rekor.server.signer.signerFileSecretOptions.secretName. | string | `""` |
 | configs.rekor.signer.secret.private_key | Private encrypted signing key | string | `""` |
 | configs.rekor.signer.secret.private_key_file | File containing a private encrypted signing key | string | `""` |
+| configs.segment_backup_job.image.pullPolicy |  | string | `"IfNotPresent"` |
+| configs.segment_backup_job.image.registry |  | string | `"quay.io"` |
+| configs.segment_backup_job.image.repository |  | string | `"redhat-user-workloads/rhtas-tenant/rhtas-stack-1-0-beta/segment-backup-job"` |
+| configs.segment_backup_job.image.version |  | string | `"sha256:d5b5f7942e898a056d2268083e2d4a45f763bce5697c0e9788d5aa0ec382cc44"` |
+| configs.segment_backup_job.name |  | string | `"segment-backup-job"` |
+| configs.segment_backup_job.namespace |  | string | `"trusted-artifact-signer-monitoring"` |
+| configs.segment_backup_job.rolebindings[0] |  | string | `"segment-backup-job"` |
+| configs.tas_monitoring.namespace |  | string | `"trusted-artifact-signer-monitoring"` |
```

Comment:
are we not going for `trusted_artifact_signer_monitoring`? I feel like this would match the pattern of our release namespace, but if you decide against it simply in the values file thats fine.

Reply:
Namespaces cannot use `_` but `-` is OK. Or maybe I misunderstand your question, @Gregory-Pereira.

Reply:
Yeah I think this is fine actually for how we reference it from the values fine.